File: 1293-three-consecutive-odds/three-consecutive-odds.py
class Solution(object):
    def threeConsecutiveOdds(self, arr):
        i = 0
        n = len(arr)
        while i < n-2 :
            if arr[i] % 2 != 0 and arr[i+1] % 2 != 0 and arr[i+2] % 2 != 0 :
                return True
            i += 1
        return False
        

File: 1293-three-consecutive-odds/three-consecutive-odds.cs
public class Solution {
    public bool ThreeConsecutiveOdds(int[] arr) {
        int i = 0;
        int n = arr.Length;
        while (i < n-2) {
            if (arr[i] % 2 != 0 && arr[i+1] % 2 != 0 && arr[i+2] % 2 != 0) 
            return true;
            i++;
        }
        
        return false;
    }
}